<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_info</class>
 <widget class="QWidget" name="edit_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1565</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(0, 0, 0);

}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>591</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(255, 0, 255);
	font: 700 48pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>Edit Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(255, 0, 255);
font: 700 25pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>Please  choose  feild which you want to change :</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>230</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: 700 25pt &quot;Script MT&quot;;
	background-color: rgb(170, 0, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="choose_box">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>829</width>
     <height>243</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
	background-color: rgb(238, 151, 255);
	color: rgb(0, 0, 0);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="last_name">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
	background-color: rgb(238, 151, 255);
	color: rgb(0, 0, 0);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
    </property>
    <property name="text">
     <string>lastname</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
	background-color: rgb(238, 151, 255);
	color: rgb(0, 0, 0);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="username">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
	background-color: rgb(238, 151, 255);
	color: rgb(0, 0, 0);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="phone_num">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
	background-color: rgb(238, 151, 255);
	color: rgb(0, 0, 0);
	font: 700 20pt &quot;Script MT&quot;;
}</string>
    </property>
    <property name="text">
     <string>phone</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="edit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>230</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(148, 0, 148);
	color: rgb(238, 238, 238);
	font: 700 30pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>469</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(255, 0, 255);
font: 700 25pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>please enter PreviousData : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PreviousData">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>561</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
	background-color: rgb(109, 5, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>590</y>
     <width>402</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(255, 0, 255);
font: 700 25pt &quot;Script MT&quot;;
}</string>
   </property>
   <property name="text">
    <string>please enter NewData : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewData">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>590</y>
     <width>641</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	font: 700 20pt &quot;Script MT&quot;;
	background-color: rgb(109, 5, 255);
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
